### DISCUSSION TITLE
What is Open in Current window?

### DISCUSSION THREAD

**Discussion by lamyergeier** [sienori/Tab-Session-Manager#668] [Q&A] [answered]
`Open in Current window`. Closes all windows and opens the session in a blank window.

Is this how is it intended to function?

**Comment by sienori** [comment, marked as answer]
This is one of the options for opening a session.
![image](https://user-images.githubusercontent.com/29514424/101608487-1e482400-3a49-11eb-999d-658a8e1adcd5.png)
Open the session in the current window. If the window contains tabs, they are closed. If there are other windows, they are closed.

**Reply by lamyergeier** [reply]
Then it should be renamed as Open a new session as Open in new Window should not close existing windows.
